javagen/src/main/java/com/azure/autorest/template/ProtocolSampleTemplate.java

```diff
@@ -0,0 +1,99 @@
+package com.azure.autorest.template;
+
+import com.azure.autorest.extension.base.model.codemodel.Parameter;
+import com.azure.autorest.extension.base.model.codemodel.RequestParameterLocation;
+import com.azure.autorest.model.clientmodel.ClassType;
+import com.azure.autorest.model.clientmodel.ClientMethod;
+import com.azure.autorest.model.clientmodel.ClientMethodParameter;
+import com.azure.autorest.model.clientmodel.MethodGroupClient;
+import com.azure.autorest.model.clientmodel.ProxyMethodExample;
+import com.azure.autorest.model.javamodel.JavaFile;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProtocolSampleTemplate {
```

Comment:
You didn't implement `IJavaTemplate` though you call it Template.

Reply:
Done